app/assets/javascripts/annotations.js

```diff
@@ -727,13 +759,18 @@ function newAnnotationBox(annotation) {
     e.preventDefault();
     box.find('.annotation-box').hide();
     box.find('.annotation-form').show().css('width', '100%');
+    
+    M.textareaAutoResize(box.find('#comment-textarea'));
```

Comment:
Could you do the same thing for global annotations? 

![Screenshot 2023-02-05 at 14 16 39](https://user-images.githubusercontent.com/9074856/216840038-2c542bbc-2cf8-4715-993c-26c17d5b4508.png)